jenkins-slaves/jenkins-slave-zap/README.md

```diff
@@ -0,0 +1,43 @@
+# jenkins-slave-zap
+
+Provides a docker image of the zap runtime for use as a Jenkins slave. The public docker registry version of OWASP's Zed Attack Proxy (ZAP) is not compatible with OpenShift without using privleged containers. This Docker image resolves that issue.
+
+## Build local
+
+`docker build -t jenkins-slave-zap .`
+
+## Run local
+
+For local running and experimentation run `docker run -i -t jenkins-slave-zap /bin/bash` and have a play once inside the container. To check the zap runtime run `/zap/zap-baseline -r index.html -t http//<url-to-test>`
+
+## Build in OpenShift
+
+```bash
+oc process -f ../templates/jenkins-slave-generic-template.yml \
+    -p NAME=jenkins-slave-zap \
+    -p SOURCE_CONTEXT_DIR=jenkins-slaves/jenkins-slave-zap \
+    -p BUILDER_IMAGE_NAME=centos:centos7 \
+    | oc create -f -
```

Comment:
You need also: `-p DOCKERFILE_PATH=Dockerfile` because the jenkins-slave-generic-template.yml assumes that dockerfile is called "Dockerfile.rhel7" (see: https://github.com/redhat-cop/containers-quickstarts/blob/master/jenkins-slaves/templates/jenkins-slave-generic-template.yml#L78)